bin/pycbc_bank_verification

```diff
@@ -36,6 +36,7 @@ import os, sys
 import copy
 import logging
 import numpy
+import h5py
 from glue.ligolw import ligolw, table, lsctables, utils as ligolw_utils
```

Comment:
Do we still need all these glue imports after your change? I'd love to drop them.